content-src/lib/selectLayoutRender.js

```diff
@@ -63,6 +68,10 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
     filterArray.push("TopSites");
   }
 
+  if (!lang.startsWith("en-")) {
+    filterArray.push("Navigation");
```

Comment:
Right now this is hard coded in here, as the component doesn't support non english right now. For the time being it's a 1 off.